src/Nethermind/Nethermind.Init/Steps/InitRlp.cs

```diff
@@ -33,6 +34,11 @@ public virtual Task Execute(CancellationToken _)
                 Rlp.RegisterDecoders(assembly);
             }
 
+            foreach (INethermindPlugin plugin in _api.Plugins)
```

Comment:
Potentially plugin order is important.

Also would we override decoder even if plugin is disabled?

Reply:
They are ordered according to PluginConfig.PluginOrder.

Yeah, it will override even when it's not required. I'll add one more `InitEncoding` method and let plugins do the work(but such method will be called before `Init`)